### PR TITLE
feat: add 财联社电报分类

### LINKED ISSUE #5875: Rss Wanted【财联社- 看盘】希望按照时间顺序推送，谢谢
### 网站地址
[https://www.cls.cn/telegraph](https://www.cls.cn/telegraph) 电报默认是全部，分列：公告，解读，加红，推送，提醒，看盘，基金。

格式如下：https://www.cls.cn/v1/roll/get_roll_list?app=CailianpressWeb&category=标志&last_time=1596675028&os=web&refresh_type=1&rn=20&sv=7.2.2&sign=hidden

|  category   | 标志  |
|  ----  | ----  |
| 公告  | announcement |
| 解读  | explain |
| 加红  | red |
| 推送  | jpush |
| 提醒  | remind |
| 看盘  | watch |
| 基金  | fund |

xhr中都包含 "subject_id":1103,"subject_name":"A股盘面直播"

### 网站描述
无
### 需要生成什么内容？
希望按照时间顺序推送 看盘的栏目信息
### 谢谢

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #5875

## 完整路由地址 / Example for the proposed route(s)

```routes
/cls/telegraph/watch
/cls/telegraph/announcement
/cls/telegraph/explain
/cls/telegraph/red
/cls/telegraph/jpush
/cls/telegraph/remind
/cls/telegraph/fund
```

## 新RSS检查列表 / New RSS Script Checklist

- [x] 这个PR中包含了新的路由吗? Does this PR add new route?
  - 如果有, 请完成检查列表. If yes, please finish the check list
  - **如果你的PR符合下方某个事项, 也请注明. If any of the checklist item meets your PR, please fill it out.**
  - [x] <- 这样打勾
- [x] 是否提供了文档? Documentation provided?
  - [ ] 是否提供了英文文档? EN Documentation provided?
- [ ] 是否支持全文获取? Is this RSS Script support fulltext?
  - [ ] 如果全文获取中需要访问文章链接, 是否使用了缓存? If fulltext requires to fetch detail pages, is cache used in the process?
  - [缓存说明](https://docs.rsshub.app/joinus/#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun) | [How to use cache](https://docs.rsshub.app/joinus/#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun)
- [ ] 目标是否有明显的反爬/频率限制? Is there any sign of anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? (延长缓存时间, 写文档说明, etc.) If yes, do your code reflect this sign? (e.g. write documentations, use long cache time)
- [ ] 是否引入的新的包? Any new package introduced?
  - 如果有, 请说明原因. If yes, please state your reason
- [ ] 是否使用了`Puppeteer`? Make use of `Puppeteer`?
  - 如果有, 请说明原因. If yes, please state your reason

### CODE REVIEW

File: docs/finance.md [hunk 1]
```diff
@@ -30,11 +30,17 @@ pageClass: routes
 
 ### 电报
 
-<Route author="nczitzk" example="/cls/telegraph" path="/cls/telegraph"/>
+<Route author="nczitzk" example="/cls/telegraph" path="/cls/telegraph/:category?" :paramsDesc="['分类，见下表']">
```
Comment (NeverBehave) [inline docs/finance.md:33]: Please include all route added in the route block.

**Review by NeverBehave — CHANGES_REQUESTED** [review]

File: docs/finance.md (hunk 1 above)
Comment (nczitzk) [reply docs/finance.md:33]: 👌

**Comment by NeverBehave** [comment]
居然已经删掉了吗？

**Comment by nczitzk** [comment]
😭

**Pull request closed without merging**